Parser/pegen/peg_api.c

```diff
@@ -4,25 +4,35 @@
 #include "pegen.h"
 
 mod_ty
-PyPegen_ASTFromString(const char *str, PyArena *arena)
+PyPegen_ASTFromString(const char *str, PyObject *filename_ob, PyArena *arena)
```

Comment:
Remind me, what's the ownership issue that requires the caller to pass the filename? Can't this function make the `PyUnicode_FromString("<string>")` call? (And ditto for the next function.)

Reply:
Once more, I just did this like this just to be consistent with `PyParser_ASTFromStringObject`.

Reply:
I think the situation is a little different -- those APIs are constrained by the need to first create the Concrete Syntax Tree and then convert that to the parse tree. Let's do the simplest thing that works for our use cases for now.

Reply:
Alright!